spec/bourbon/functions/modular_scale_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "spec_helper";
+
+describe "modular-scale" do
+  before(:all) do
+    ParserSupport.parse_file('functions/modular-scale')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.